Header header
float32 drone_timeuint16 tag
uint16 size
uint16[] raw_accs
int16[] raw_gyros
int16[] raw_gyros_110
uint32 vbat_raw
uint16 us_debut_echo
uint16 us_fin_echo
uint16 us_association_echo
uint16 us_distance_echo
uint16 us_courbe_temps
uint16 us_courbe_valeur
uint16 us_courbe_ref
uint16 flag_echo_ini
uint16 frame_number
uint16 nb_echo
uint32 sum_echo
int32 alt_temp_raw
int16 gradient
